### ISSUE TITLE
Persistent SSL certificate issues 

### ISSUE THREAD

**Issue by RobPasMue** [ansys/python-installer-qt-gui#93]
Follow up from #83

Hi @RobPasMue 
Thanks for quick fix.
1. This works for downloading now, I cannot install Python because of `Admin Rights`. That is fine for now.
2. I have also encountered  SSL problem with updater 

![grafik](https://user-images.githubusercontent.com/82146655/235189287-b5c4ba97-60c0-42a1-97e6-655ff6ae371b.png)

I am not really sure where these SSL problems are arising from exactly, because I can access everything over the browser without any problems.

_Originally posted by @kmahajan-cadfem in https://github.com/pyansys/python-installer-qt-gui/issues/83#issuecomment-1527736318_

**Comment by RobPasMue** [comment]
Hi @kmahajan-cadfem - could you verify that this issue is still occuring on your side? Thanks in advance!

**Comment by kmahajan-cadfem** [comment]
@RobPasMue, this is indeed the case and I think this is more related to the network config(reverse proxy and cert managers) on our side, since I cannot access the some websites(including Github) using `requests` or `pygithub` from python, irrespective of the `Python Manager`. 
 so if you have totally offline more in your mind, I dont think this could be resolved just using the code from `Python Manager`.

**Comment by RobPasMue** [comment]
Yes, that was my feeling as well... Let's try something though, it might be only related to the fact that you can't verify the certificate. Could you try running the following code on your side and see if it works?

You will need to install ``pygithub`` (``requests`` will be installed as well, since it is a dependency)

```py
import github as gh
import requests

# Disable SSL certificate verification
session = requests.Session()
session.verify = False

gh = Github()
repo = gh.get_repo(f"ansys/python-installer-qt-gui")

# Get the latest release and its tag name
latest_release = repo.get_latest_release()
latest_version_tag = latest_release.tag_name

print(latest_version_tag)

# Re-enable SSL verification
session.verify = True

```

**Comment by RobPasMue** [comment]
What I could do (otherwise) is to show a URL to the latest release so that users can verify manually whether they are in the last version or not (in case the Python Manager code fails using ``requests``). That is definitely feasible. =) Won't be automated - but it will allow you to easily verify it at least

**Comment by kmahajan-cadfem** [comment]
> Yes, that was my feeling as well... Let's try something though, it might be only related to the fact that you can't verify the certificate. Could you try running the following code on your side and see if it works?
> 
> You will need to install `pygithub` (`requests` will be installed as well, since it is a dependency)
> 
> ```python
> import github as gh
> import requests
> 
> # Disable SSL certificate verification
> session = requests.Session()
> session.verify = False
> 
> gh = Github()
> repo = gh.get_repo("ansys/python-installer-qt-gui")
> 
> # Get the latest release and its tag name
> latest_release = repo.get_latest_release()
> latest_version_tag = latest_release.tag_name
> 
> print(latest_version_tag)
> 
> # Re-enable SSL verification
> session.verify = True
> ```

Did not work ...

**Comment by kmahajan-cadfem** [comment]
> What I could do (otherwise) is to show a URL to the latest release so that users can verify manually whether they are in the last version or not (in case the Python Manager code fails using `requests`). That is definitely feasible. =) Won't be automated - but it will allow you to easily verify it at least


This would be great, so that user can see for himself, if he wants new version or not.

**Comment by RobPasMue** [comment]
Implemented in #152 - have a look at it @kmahajan-cadfem and let me know what you think. Thanks for the feedback!
